airbyte-cdk/python/airbyte_cdk/sources/streams/__init__.py

```diff
@@ -0,0 +1,4 @@
+# Initialize Streams Package
+from .core import Stream
```

Comment:
Curious - what is the reason for using relative imports? I noticed we were previously using absolutes. https://github.com/airbytehq/airbyte/blob/master/airbyte-cdk/python/airbyte_cdk/base_python/__init__.py#L32